docs/world api.md

```diff
@@ -455,7 +456,7 @@ In addition, the following methods can be implemented and attributes can be set
 ```python
 def generate_early(self) -> None:
     # read player settings to world instance
-    self.final_boss_hp = self.world.final_boss_hp[self.player].value
+    self.final_boss_hp = self.world.options["final_boss_hp"].value
```

Comment:
similar to previous comment: How does it know which player's `final_boss_hp` to get the value from?

Reply:
These dictionaries exist per `World` so this dictionary is all of the options that your current world's player chose.

Reply:
What is the `self` that has a `world` on it? This doesn't include the PR that changes `world` to `multiworld`, right?

I think that's a `MultiWorld` that the documentation is referencing.

Reply:
it's a typo :)